wetterdienst/service.py

```diff
@@ -120,6 +120,7 @@ def dwd_stations(
 @app.get("/api/dwd/{product}/values")
 def dwd_values(
     product: str,
+    tabular: bool = Query(default=False),
```

Comment:
Should this be tidy?

Reply:
Sure. I amended the parameter to say `tidy: bool = Query(default=True)`.